srv/salt/ceph/configuration/files/rgw.conf

```diff
@@ -1,4 +1,4 @@
 [client.{{ client }}]
-rgw frontends = "beast endpoint=[::]:80"
+rgw frontends = "beast endpoint=[::]:{{ pillar.get('rgw_frontend_port', 80) }}"
```

Comment:
`"beast port={{ pillar.get('rgw_frontend_port', 80) }}"`

In wake of https://github.com/SUSE/ceph/pull/295 we need to use port as that will bind to both ipv4 and v6

Reply:
done